site.yml

```diff
@@ -40,3 +40,5 @@ asciidoc:
     idprefix: ''
     idseparator: '-'
     oc-contact-url: https://owncloud.com/contact/
+    php-supported-versions-url: https://secure.php.net/supported-versions.php
```

Comment:
Why this variable? This is the same for all versions of the docs.

Reply:
Because some attributes are applicable to the entire project, such as this one, whereas others are only applicable to an individual manual or page. To me, this one is applicable across all manuals.